webui/react/src/pages/ModelRegistry.tsx

```diff
@@ -413,13 +430,24 @@ const ModelRegistry: React.FC<Props> = ({ workspace }: Props) => {
         { key: MenuKey.SwitchArchived, label: record.archived ? 'Unarchive' : 'Archive' },
       ];
 
+      if (canMoveModel({ destination: { id: workspace?.id ?? 1 } })) {
+        menuItems.push({ key: MenuKey.MoveModel, label: 'Move' });
+      }
       if (user?.id === record.userId || user?.isAdmin) {
```

Comment:
this should be changed to `canDeleteModel({ model: record })` for RBAC; this removes `user?.id` and `user?.isAdmin` from the dependencies

Reply:
We can ignore this until 6016